tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py

```diff
@@ -23,16 +23,13 @@
 from tensorflow_model_optimization.python.core.clustering.keras import cluster
 from tensorflow_model_optimization.python.core.clustering.keras import cluster_wrapper
 from tensorflow_model_optimization.python.core.clustering.keras import clusterable_layer
-from tensorflow_model_optimization.python.core.clustering.keras import clustering_registry
```

Comment:
Changes in this file seem unrelated to the topic. If it is a clean-up, consider moving them to a separate PR.

Reply:
It is indeed a leftover from a cleanup. Will move it to a separate change.